gpjax/kernels.py

```diff
@@ -59,20 +59,10 @@ def ard(self):
         """
         return True if self.ndims > 1 else False
 
-    @property
-    def params(self) -> dict:
+    def _initialise_params(self, key: jnp.DeviceArray) -> Dict:
         """A template dictionary of the kernel's parameter set."""
         return self._params
```

Comment:
We need to have a think about `._params` on the Kernel object. Firstly, since the `.params` property method no longer exists, we could replace `_params`  with `params` when defining a Kernel.

But the bigger issue is that we should either adopt the same convention for likelihoods and mean functions, or work to remove it. It is important that all objects in GPJax behave the same.